lib/puppet/type/scheduled_task.rb

```diff
@@ -92,32 +96,33 @@ def insync?(current)
       A trigger can contain the following keys:
 
       * For all triggers:
-          * `schedule` **(Required)** --- What kind of trigger this is.
-            Valid values are `daily`, `weekly`, `monthly`, or `once`. Each kind
-            of trigger is configured with a different set of keys; see the
-            sections below. (`once` triggers only need a start time/date.)
+          * `schedule` **(Required)** --- The schedule type. Valid values are
+            `daily`, `weekly`, `monthly`, or `once`.
```

Comment:
I don't think we want to make these changes, beyond just adding the `daily` trigger below. @nfagerlund and I spent some time coming up with "kind of trigger" because "type" is an overloaded term in puppet.